docker/docker-compose.yaml

```diff
@@ -17,11 +17,11 @@ services:
     env_file:
       - ../.env
 
-  db:
+  db1:
```

Comment:
Why was this renamed to `db1`?  I think it should be OK to leave it as `db`.

Reply:
I hit the same issue of ``` ERROR: for db  Cannot create container for service db: Conflict. The container name "/kernelci-db" is already in use ```. So I tried to rename the service to db1 and check.
It was for testing only.
Yes, we don't need to rename the service name.